docs/ops/arithmetic/SquaredDifference_1.md

```diff
@@ -4,7 +4,14 @@
 
 **Category**: Arithmetic binary operation
 
-**Short description**: *SquaredDifference* performs element-wise subtraction operation with two given tensors applying multi-directional broadcast rules, after that each result of the subtraction is squared.
+**Short description**: *SquaredDifference* performs element-wise subtract and square the result operation with two given tensors applying multi-directional broadcast rules.
```

Comment:
`... applying broadcasting rule specified in the *auto_broadcast* attribute.`. The same comment for other specification files

Reply:
So basically you want me to repeat what is said in detailed description. What's the point ?:)